spec/helpers/application_helper_spec.rb

```diff
@@ -46,4 +46,30 @@
       expect(helper.render_markdown('**abc**')).to include content_tag(:strong, 'abc')
     end
   end
+
+  describe '#render_user_information' do
+    before do
+      helper.extend(Module.new do
+        def current_user; end
+      end)
+
+      allow(helper).to receive(:current_user).and_return(user)
+    end
+
+    let(:user) do
+      User.new name: 'Some Body', webauth: 'xyz'
```

Comment:
We could use `build(:webauth_user, name: 'Some Body')` to take advantage of `FactoryGirl`.